tools/releaseBuild/Images/microsoft_powershell_windowsservercore/wix.psm1

```diff
@@ -4,14 +4,18 @@ Import-Module "$PSScriptRoot\dockerInstall.psm1"
 
 # Install using Wix Zip because the MSI requires an older version of dotnet
 # which was large and unstable in docker
-function Install-WixZip
+function Install-Wix
 {
-    param($zipPath, $arm64 = $false)
+    param($arm64 = $false)
 
     $targetRoot = $arm64 ? "${env:ProgramFiles(x86)}\Arm Support WiX Toolset xcopy" : "${env:ProgramFiles(x86)}\WiX Toolset xcopy"
+    # cleanup previous install
+    if(Test-Path $targetRoot) {
+        Remove-Item $targetRoot -Recurse -Force
+    }
     $binPath = Join-Path -Path $targetRoot -ChildPath 'bin'
-    Write-Verbose "Expanding $zipPath to $binPath ..." -Verbose
-    Expand-Archive -Path $zipPath -DestinationPath $binPath -Force
+    Register-PSRepository -Name NuGetGallery -SourceLocation https://api.nuget.org/v3/index.json
```

Comment:
We need to use https://pkgs.dev.azure.com/powershell/PowerShell/_packaging/powershell/nuget/v3/index.json or https://pkgs.dev.azure.com/powershell/PowerShell/_packaging/powershell/nuget/v2

also, we should have csproj so compliance systems knows that we use the package.